doc/classes/Color.xml

```diff
@@ -300,295 +300,295 @@
 	<constants>
 		<constant name="gray" value="Color( 0.75, 0.75, 0.75, 1 )">
 		</constant>
-		<constant name="aliceblue" value="Color( 0.94, 0.97, 1, 1 )">
+		<constant name="darkorchid" value="Color( 0.6, 0.2, 0.8, 1 )">
```

Comment:
These changes are weird, `doctool` should normally order things in the same order that they are registered in the source code (which was the case previously).

What OS did you run it on? Sounds like it's not doing reproducible output based on the platform.

Reply:
This was generated on Windows 10, using the Visual Studio developer terminal.